E/Eiscor/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Eiscor"
+version = v"0.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/eiscor/eiscor.git", "696b90afcd8d637d080fe29de5c3c799bf5eb8dd")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd eiscor-0.2.0/
+export HOME=$libdir
```

Comment:
This is just to set the install directory. See https://github.com/eiscor/eiscor/blob/849f56c64eb79a4fd75c46c9d2591045a6d5c889/make.inc#L8

Reply:
I think you should be able to instead do `make INSTALLDIR=$libdir ...`